vaadin-grid-pro-flow-parent/vaadin-grid-pro-flow/src/main/java/com/vaadin/flow/component/gridpro/GridPro.java

```diff
@@ -172,6 +174,22 @@ public EditColumn(GridPro<T> grid, String columnId,
             super(grid, columnId, renderer);
         }
 
+        @Override
+        protected void onAttach(AttachEvent attachEvent) {
+            super.onAttach(attachEvent);
+            // Detaching GridPro will clear editor field reference from client,
+            // re-register it.
+            if (getEditorType().equals("custom")) {
+                setEditModeRenderer(editorField);
+            }
+        }
+
+        private void setEditModeRenderer(AbstractField component) {
```

Comment:
It's now a duplicate of `setEditModeRenderer` available in `EditColumnConfigurator`.
I wonder if it is possible to avoid duplicating it 🤔

Reply:
I made setEditModeRenderer  in column package protected to get rid of setEditModeRenderer in EditColumnConfigurator

Reply:
I have just tried to use `addAttachListener` in `EditColumnConfigurator` and removed all changed to `GridPro`. It seems that all tests are passing, so I guess there is a way to fix it within `EditColumnConfigurator` without moving `setEditModeRenderer` and making it protected.
The code I have tested, but it could be simplified (inside `EditColumnConfigurator.custom`):
```
if (attachRegistration != null) {
    attachRegistration.remove();
    attachRegistration = null;
}
attachRegistration = column.getElement()
        .addAttachListener(e -> setEditModeRenderer(component));

column.getElement().getNode()
        .runWhenAttached(ui -> ui.beforeClientResponse(column,
                context -> setEditModeRenderer(component)));
```

Reply:
Fixed the PR